asset_dashboard/templates/asset_dashboard/partials/forms/add_edit_phase_form.html

```diff
@@ -162,6 +185,38 @@ <h3>Phase Assets</h3>
   <script type="text/javascript">
     window.props = JSON.parse(document.getElementById('props').textContent)
     window.reactMount = document.getElementById('map')
+    // Don't worry about auto-saving or funding stream forms if it's a new phase
+    if (!`{{ phase }}`) {
+      throw new Error("This is a new phase and will need to be saved manually")
+    }
+
+    $(".icon-loader, .icon-failed").hide()
+
+    let delayTimer
+    function trackChanges() {
+      // All visible inputs will trigger a save if any they detect any changes
+      const allInputs = $(":input:not(.btn.btn-info, :hidden)")
```

Comment:
I'm grabbing all inputs every time this runs so I can re-run this later on and include new funding forms as they get added